R/aggregate.R

```diff
@@ -12,34 +12,39 @@
 #' @param geoweights_table A table of weights which can be generated using
 #'   the function calc_geoweights()
 #' @param second_weights A boolean indicating whether secondary weights should be used
+#'
+#' @examples
+#' staggregate_polynomial(demo_prcp, "prcp", daily_agg = "sum", degree = 3, geoweights_table = demo_output_geoweights, second_weights = TRUE)
+#' staggregate_spline(demo_prcp, "prcp", daily_agg = "sum", knot_locs = c(1,2,3), geoweights_table = demo_output_geoweights, second_weights = TRUE)
+#' staggregate_bin(demo_prcp, "prcp", daily_agg = "sum", num_bins = 30, min = 5, geoweights_table = demo_output_geoweights, second_weights = TRUE)
+#'
+#'
 
 ## Sara Orofino
```

Comment:
We should probably remove personal information like this from the code comments? This chunk of comments seems unnecessary ?